src/app/utils/chart-utils.ts

```diff
@@ -10,8 +10,8 @@ export enum ChartDuration {
 
 export const durationToQueryParams = {
   [ChartDuration.TODAY]: {
-    bucket_size_seconds: 60 * 60,
-    limit: 24,
+    bucket_size_seconds: 60 * 5,
+    limit: (60 / 5) * 24, // full day for 5 minutes buckets
```

Comment:
@lubej this is changing defaults you created. I can move it to ActiveAccounts if you want, because this will affect your tx chart work. API needs 300 for 5 mins buckets. And we need fetch full day of 5mins buckets. For active accounts we need to use every 12th item (or sum groups of 12, in case of tx_volume)

Reply:
Disregard for now. Can be merged in, as is.